server/cpho/schema.py

```diff
@@ -100,7 +100,7 @@ def resolve_indicator(root, info, **kwargs):
 class CreateIndicator(graphene.Mutation):
     class Arguments:
         category = graphene.String(required=True)
-        sub_category = graphene.String(required=True)
```

Comment:
no need to touch the graphql stuff right now, it's all being re-written

Reply:
GraphQL stuff is still pretty much the same except for the subcategory field that got changed to topic.
ShouldI checkout schema.py from main to revert the changes? or keep it as it is?

Reply:
I guess it doesn't matter, it'll be a full rewrite anyway